.gitpod.yml

```diff
@@ -0,0 +1,30 @@
+github:
+  prebuilds:
+    master: true
+    branches: true
+    pullRequests: true
+    pullRequestsFromForks: true
+    addCheck: false
+    addComment: false
+    addBadge: true
+
+tasks:
+  - name: Start web server
+    command: python3 -m http.server --directory examples/wasm
+
+  - name: Install linux dependencies
+    before: |
+      sudo apt-get install --no-install-recommends -yq libasound2-dev libudev-dev
+
+  - name: Install wasm tooling
+    before: |
+      rustup target add wasm32-unknown-unknown
+      cargo install cargo-quickinstall
+      cargo quickinstall wasm-bindgen-cli
+      cargo quickinstall cargo-watch
+    init: |
+      cargo run -p build-wasm-example -- lighting
```

Comment:
Do we want to pre-suppose which example is relevant. It would be very cool, for example, to select a relevant example which has been updated by the given PR. Or at least to have a dropdown with all the examples.

OTOH, this might be the best possible option within the technical constraints of the gitpod platform.

Reply:
The lighting example is one where a lot of things could go wrong, so it's a good first pick. My goal was mostly to cache everything for a wasm build, so why not go all the way to build an example